### ISSUE TITLE
Performs convolutions over the embedded word vectors using multiple filter sizes

### ISSUE THREAD

**Issue by zzyxzz** [deeplearning4j/deeplearning4j#1088]
Something similar to
http://www.wildml.com/2015/12/implementing-a-cnn-for-text-classification-in-tensorflow/
and the paper "Convolutional Neural Networks for Sentence Classification" link
http://arxiv.org/abs/1408.5882

Some descriptions to make it more clear.
"The first layers embeds words into low-dimensional vectors. The next layer performs convolutions over the embedded word vectors using multiple filter sizes. For example, sliding over 3, 4 or 5 words at a time. Next, we max-pool the result of the convolutional layer into a long feature vector, add dropout regularization, and classify the result using a softmax layer."

Thanks.

**Comment by AlexDBlack** [comment]
Just a heads up: this sort of architecture should be relatively straightforward with the computation graph architecture we've been working on. The idea with that is that you define a network using an arbitrary directed acyclic graph connection structure, instead of just a stack of layers as in MultiLayerNetwork.
This should be merged to master soon.

**Comment by agibsonccc** [comment]
Right. That's what I was mentioning earlier. Mainly flagging for later.

**Comment by nyghtowl** [comment]
Addressed with comp graph

**Comment by ahmetaker** [comment]
Hi,
is there any code example for this particular problem you could share?
thanks,
ahmet
